accountAPI.yaml

```diff
@@ -398,11 +399,20 @@ components:
           $ref: '#/components/schemas/commonAccountId'
         account:
           $ref: '#/components/schemas/paymentIbanAccount'
+        accountOwner:
+          maxLength: 140
```

Comment:
accountOwner in bLink /Consent endpoint is limited with maxLength: 70. 
Generally, i agree on 140 as it complies to other free text fields, but difference should be considered and in long-term also adjsted in bLink /consents.

Reply:
maxLength will be set to 140 to be in line with the other free text fields.